web/src/app/dialogs/FormDialog.js

```diff
@@ -48,18 +47,23 @@ function FormDialog(props) {
   const classes = useStyles()
   const width = useWidth()
   const isWideScreen = isWidthUp('md', width)
+  const [open, setOpen] = useState(false)
+
+  useEffect(() => {
+    setOpen(true)
+  }, [])
+
   const {
     alert,
     confirm,
     disableGutters,
     errors,
     fullScreen,
-    isUnmounting,
     loading,
     primaryActionLabel, // remove from dialogProps spread
     maxWidth,
     notices,
-    onClose,
+    onClose: onExited,
```

Comment:
Since this is only used in one place, let's leave it as the prop name, and remove the wrapper around `setOpen` (less indirection).

```suggestion
    onClose,
```

Further down we can do the switch

```js
<Dialog
 // ...

// only calling onClose after the exit event so that animations finish properly
onClose={()=>setOpen(false)}
onExited={onClose}
/>
```

Otherwise looking at the code below `onClose` may seem like it's the `onClose` prop rather than a thin wrapper and `onExited` being the actual `onClose` prop. If we handle things directly using original names there is no misinterpretation.

Reply:
In that case, let's call them `handleOnExited` and `handleOnClose` to denote that they are internal handlers and not the `on<event>` props passed in